__tests__/intergration.test.js

```diff
@@ -70,12 +70,51 @@ describe("GET /api", () => {
     test("each API property should be correct type", () => {
         return request(app).get("/api").expect(200).then(result => {
             const returnObj = result.body.API;
-            console.log(returnObj);
             for (const key in returnObj) {
                 expect(typeof returnObj[key].description).toBe("string")
                 expect(Array.isArray(returnObj[key].queries)).toBe(true)
                 expect(typeof returnObj[key].exampleResponse).toBe("object")
             }
         })
     })
-});
\ No newline at end of file
+});
+describe("get all articles endpoint", () => {
+    test("Should return a 200 status code", () => {
+        return request(app).get("/api/articles/").expect(200);
+    })
+    test("Should return an array in property 'articles'", () => {
+        return request(app).get("/api/articles/").expect(200).then(result => {
+            expect(result.body).hasOwnProperty("articles");
+            expect(Array.isArray(result.body.articles)).toBe(true);
+        })
+    })
+    test("Should return each article with correct properties", () => {
+        return request(app).get("/api/articles/").expect(200).then(result => {
+            const resultArray = result.body.articles;
+            resultArray.forEach(article => {
```

Comment:
make sure you always check the length of the array before running a .forEAch - it can produce false positives